src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/Address.java

```diff
@@ -153,14 +153,23 @@ public void setValidUntil(LocalDate validUntil) {
 
     @JsonIgnore
     public AddressType getAddressType() {
-        if (Objects.nonNull(this.getValidUntil()) && isPastDate(this.getValidUntil())) {
-            return AddressType.PREVIOUS;
+
+        if (Objects.isNull(this.getValidUntil())) {
+            if (Objects.isNull(this.getValidFrom())) {
+                return AddressType.CURRENT;
+            } else if (isPastDateOrToday(this.getValidFrom())) {
+                return AddressType.CURRENT;
+            } else {
```

Comment:
this final else statement can be deleted. 
The `return null;` on line 175 should suffice ( please check that it does )